integration-test-groups/cxf-soap/cxf-soap-server/src/test/java/org/apache/camel/quarkus/component/cxf/soap/it/CxfSoapServiceTest.java

```diff
@@ -38,7 +38,7 @@ public void simpleSoapService() {
         final HelloPortType helloPort = service.getHelloPort();
         String endpointURL = getServerUrl() + "/soapservice/hello";
         ((BindingProvider) helloPort).getRequestContext().put(BindingProvider.ENDPOINT_ADDRESS_PROPERTY, endpointURL);
-        Assertions.assertThat(helloPort.hello("World")).isEqualTo("Hello World from CXF service");
```

Comment:
We could have kept using assert-j it tends to have nicer error messages. Not worth bothering with it right now, let's keep it as you have it.

Reply:
thanks for noticing!